README.md

```diff
@@ -1,2 +1,74 @@
-# rfcs
-RFCs for changes to Google Cartographer projects
+# Cartographer RFCs
+[Cartographer RFCs]: #cartographer-rfcs
+
+For changes more substantial than simple bugfixes and documentation improvements
+we ask that these be put through a very lightweight design review process.
+
+"RFC" stands for "request for comments" and is used by many projects, e.g.
+rust-lang, to establish a consistent and controlled path for new features to
+enter their project.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Introduction](#cartographer-rfcs)
+  - [Table of Contents]
+  - [When should you write an RFC]
+  - [Motivation for RFCs]
+  - [Before creating an RFC]
+  - [What the process is]
+
+## When should you write an RFC
+[When should you write an RFC]: #when-should-you-write-an-rfc
+
+You should provide an RFC if you intent to make a "significant" change or
```

Comment:
Use \* ?
*significant*

Reply:
`significant change or contribution` sounds ambiguous to me. How about `API breaking change or a new feature development`?

Reply:
new feature development is also ambigious though... Also API is not stable yet, so everything is a breaking change. Or nothing.

I suggest we leave the "significant" and replace it when we have a better understand of how RFCs pan out.

Reply:
Clarified.